packages/@adobe/spectrum-css-temp/components/tags/index.css

```diff
@@ -18,12 +18,20 @@ governing permissions and limitations under the License.
 
 .spectrum-Tags {
   display: inline;
+
+  &.focus-ring {
+    outline: none;
+    box-shadow: 0 0 0 2px var(--spectrum-tag-border-color-key-focus);
+    display: block;
```

Comment:
because we need to use a block level display for box-shadow, maybe it'd be better to have spectrum-Tags display as `inline-block`? otherwise, swapping between inline and block can drastically affect layout, better to keep it the same at all times. Another option is to put a display block level element (ie div) in between tag and the tags themselves

also, I can still see an outline if I tab to the group, then click on it. seems to be a result of focus-visible
![Screenshot 2023-04-10 at 6 24 24 PM](https://user-images.githubusercontent.com/698229/231031084-b52f4a3a-a396-4612-a78c-e08e8607df46.png)
I do not see this issue with ListView for some reason, so could start by comparing between the two